src/main/python/camdkit/framework.py

```diff
@@ -364,6 +773,32 @@ def make_documentation(cls) -> dict:
         "description" : desc.__doc__,
         "constraints" : desc.validate.__doc__,
         "sampling" : str(desc.sampling.value),
-        "units": desc.units
+        "units": desc.units if hasattr(desc, "units") else "None"
```

Comment:
All items must have `units` specified in `model.py`, even `None`.

Reply:
This is not enforced currently (sampling and canonical name are) - shall I update ParameterContainer.__init_subclass__ to enforce units?